src/test/resources/BranchFilterModuleTest/exp/input.ditamap

```diff
@@ -50,14 +44,14 @@
       </ditavalref>
     </topicref>
     <topicref class="- map/topicref " href="http://example.com/install.dita" scope="external"/>
-  </topicref>
-  <topicref class="- map/topicref " href="install-win.dita" keyscope="win-install">
-    <ditavalref class="+ map/topicref ditavalref-d/ditavalref " format="ditaval" href="win.ditaval" processing-role="resource-only">
+    <ditavalref class="+ map/topicref ditavalref-d/ditavalref " format="ditaval" href="mac.ditaval" processing-role="resource-only">
       <ditavalmeta class="+ map/topicmeta ditavalref-d/ditavalmeta ">
-        <dvrResourceSuffix class="+ topic/data ditavalref-d/dvrResourceSuffix " name="dvrResourceSuffix">-win</dvrResourceSuffix>
-        <dvrKeyscopePrefix class="+ topic/data ditavalref-d/dvrKeyscopePrefix " name="dvrKeyscopePrefix">win-</dvrKeyscopePrefix>
+          <dvrResourceSuffix class="+ topic/data ditavalref-d/dvrResourceSuffix " name="dvrResourceSuffix">-mac</dvrResourceSuffix>
```

Comment:
Is this correct? `win` has been replaced with `mac`, further on, there are leading spaces added.